packages/web-components/src/components/va-search-input/va-search-input.tsx

```diff
@@ -253,6 +255,8 @@ export class VaSearchInput {
         );
         break;
       default:
+        if (event.key.length !== 1) return;
+        this.value = this.value + event.key;
```

Comment:
I don't like this approach because it requires us to code additional functionality to keep the value of `value` in sync with what the user is seeing. For example, if a user types `for`, selects `for` with mouse, presses backspace and `x`, it causes the input field value to change to `forx`.

Reply:
We would also need to handle key combinations like command+delete and any other combinations that affect the input field.